docs/qt.md

```diff
@@ -12,16 +12,55 @@ Qt5 support must be enabled in the build system.
 
 ## Qt5
 
-Qt5 is a framework primarily used for cross-platform GUI based applications
-and is freely available from [qt.io](https://qt.io) or through your
-system's package manager:
+Qt5 is a framework primarily used for cross-platform GUI based applications.
+The Qt official website as of writing is [qt.io](https://qt.io).
+Qt can be downloaded directly from there but might require a account to be
```

Comment:
It hasn't been my experience that an account (not `a account`) is required, but I had to create one to file a bug.  I didn't attempt to download prebuilt binaries or QtCreator.  I think we can skip the discussion of accounts here.

Reply:
They definitely seem to want you to create an account, but I just tried their installer and it does has an option to skip signing in, so I will remove it.